src/declaration.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*.scss';
```

Comment:
Creating this file fixes linter errors on the `scss` import statements ("cannot find module" error)